src/collective/easyform/fields.py

```diff
@@ -26,15 +25,36 @@
 from zope.schema.interfaces import IField
 
 
+@implementer(IForm)
+class GenericFormWrapper(object):
+    def __init__(self, view):
+        self.__view__ = view
+
+    def __getattr__(self, item):
+        return getattr(self.__view__, item)
+
+
 @implementer(IValidator)
 @adapter(IEasyForm, Interface, IEasyFormForm, IField, Interface)
-class FieldExtenderValidator(z3c_validator.SimpleFieldValidator):
-
+class FieldExtenderValidator(object):
```

Comment:
It seems nicer to keep the `SimpleFieldValidator` as base. You can at least skip the `__init__` then, because it is the same.

Reply:
I got rid of it to reduce the confusion as to why super wasn't being called.